README-template.md

```diff
@@ -415,7 +419,7 @@ Please note there are a few caveats:
   `INSERT INTO PARTITION OVERWRITE` statements are currently not supported. Note, however, that
   partitioning and clustering in BigQuery are supported via `TBLPROPERTIES`. See the corresponding
   sections on [partitioning](#partitioning) and [clustering](#clustering).
-* CTAS (aka `CREATE TABLE AS SELECT`).
+* CTAS (aka `CREATE TABLE AS SELECT`) is currently not supported.
```

Comment:
can we add CTLT (create table like table) not support too?

Reply:
Added